e2e/terraform/gke/README.md

```diff
@@ -0,0 +1,39 @@
+# STEPS TO CREATE GKE CLUSTER USING TERRAFORM
+
+## Step-1 : Obtain the Google Credentials from Google Developers Console 
+
+- Log into the Google Developers Console and select a project.
+- The API Manager view should be selected, click on "Credentials" on the left, then "Create credentials," and finally "Service account key."
+- Select "Compute Engine default service account" in the "Service account" dropdown, and select "JSON" as the key type.
+- Clicking "Create" will download your credentials.
+- Rename this file to account.json and place in the directory where the terraform templates are present/will-be-placed
```

Comment:
Can a screenshot be provided that shows the list of files in the particular directory?